lib/board.rb

```diff
@@ -0,0 +1,43 @@
+require 'pry'
+class Board
+  @cells = []
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+  def cells
+    @cells
+  end
+  def cells=(array)
+    @cells = array
+  end
+  def reset!
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+  def position(input)
+    @cells[input.to_i-1]
+  end
+  def full?
+    select_if_empty = @cells.select { |cell| cell == " " }
+    select_if_empty == [] ? (true) : (false)
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`select_if_empty == []`